manuscript/draft.Rmd

```diff
@@ -166,3 +177,84 @@ None of the models are true forcasts as they use observed predictor variables (e
 * Need predictions at multiple time scales
 * Prediction quality at short time scales might not be a good indicator of 
   long-term effectiveness
+
+
+
+## Box 1: Ten simple rules for making and evaluating ecological forecasts
+
+### 1. Compare multiple modeling approaches
+
+Typically ecological forecasts use one modeling approach or a small number of
+related approaches. By fitting and evaluating multiple modeling approaches we
+can learn more rapidly about the best approaches for making predictions for a
+given ecological quantity. This includes comparing process based and data-driven
+models.
+
+### 2. Use time-series data when possible
+
+Forecasts describe how systems are expected to change through time. While some
+areas of ecological forecasting focus primarily on time-series data, others
+primarily focus on using spatial models and space-for time substitutions. Using
+ecological and environmental time-series data allows the consideration of actual
+dynamics from both a process [@treddenick2016a] and error structure perspective.
+
+### 3. Pay attention to uncertainty
+
+Understanding confidence in a forecast is just as important as understanding the
```

Comment:
Also, this might be a good place to mention error propagation (i.e. our uncertainty in one step should affect any estimates made subsequent steps). Otherwise, we can put it somewhere in the main text.